crates/phactory/src/system/gk.rs

```diff
@@ -2248,11 +2150,11 @@ pub mod tests {
         let report = r.gk.egress.drain_working_info_update_event();
         assert_eq!(
             FixedPoint::from_bits(report[0].settle[0].payout),
-            fp!(14.69197867920878555043)
+            fp!(576424.184820453732333905)
```

Comment:
The entire budget is paid to the single worker here.
We need to carefully carry the reward budget from khala to phala gradually.

Reply:
That's by design. Once Phala and Khala are live at the same time, we plan to run a Phat Contract to allocate budget between the two. The allocation will be calculated based on the workers (and their scores, etc). I think that's what you mean "carry the budget gradually". Am I right?

Reply:
> Am I right?

Yes. Absolutely.